### PR TITLE
Paid feature docs

### PR DESCRIPTION
**Pull request by mazameli**
Includes:
- White labeling docs
- SAML + JWT auth docs
- Data sandboxing docs
- Changes to admin guide TOC

Remaining things:

- [x] ~I'd love for these paid feature articles to include a link to a specific store page or whatever to give users info about how to get these features once we know what that URL(s) is.~ We'll add this in the future.
- [x] ~I've added `(paid feature)` to the titles of these articles in the TOC, which is a little inelegant. Have we settled on the nomenclature for this? "Commercial?" "Premium?"~ Replaced with an asterisk next to each of these, with a note at the bottom of the TOC that these are available in the enterprise edition. 
- [x] The only additional thing I might want to add to the sandboxing docs is an example of a more complex data sandboxing setup that uses a saved question; the example I wrote up just uses the simple attributes-only sandboxing flow.

### CODE REVIEW

File: docs/administration-guide/start.md [hunk 1]
```diff
@@ -2,22 +2,33 @@
 
 Are you in charge of managing Metabase for your organization? Then you're in the right spot. You are the chosen one.
 
-**This guide will teach you about:**
```
Comment (salsakran) [inline docs/administration-guide/start.md]: This re-ordering will muck with the "next" link at the bottom of each page.
Do we still want a linear path through the guide or should we just have a topic index?

File: docs/administration-guide/16-authenticating-with-saml.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+## Authenticating with SAML
```
Comment (salsakran) [inline docs/administration-guide/16-authenticating-with-saml.md]: we should also write up docs for JWT

File: docs/administration-guide/start.md (hunk 1 above)
Comment (mazameli) [reply docs/administration-guide/start.md]: Yeah, I really would prefer to go to a navigable index, accessible from each article. But I'm concerned that that will be a time suck, so I'm thinking for now I might just fix each article's "next" link to follow the new order.

File: docs/administration-guide/16-authenticating-with-saml.md (hunk 2 above)
Comment (mazameli) [reply docs/administration-guide/16-authenticating-with-saml.md]: Yep, going to push JWT docs up in a minute.

**Comment by mazameli** [comment]
Okay, I went ahead and wrote up examples for advanced sandboxing using a filtering question and added those in.

This PR should now be ready for final review.

**Pull request merged**